lib/elixir/lib/function.ex

```diff
@@ -2,10 +2,10 @@ defmodule Function do
   @moduledoc """
   A set of functions for working with functions.
 
-  We can have two types of captured functions: external and local.
+  There are two types of captured functions: **external** and **local**.
```

Comment:
s/external/remote?

Reply:
This is the function internal naming. So we need to stick with external.

Reply:
TIL, thanks.

Reply:
Yes, we should stick with "external" - it's what `:erlang.fun_info(fun, :type)` returns. Although, internally in erts they are called exports.